app/controllers/shop/items_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Shop::ItemsController < ApplicationController
+  def show
+    @item = find_item
```

Comment:
It lives in the application controller, as a result of some refactoring we did towards the end

def find_item
    Item.find(params[:id])
 end